src/components/RouteWrapper.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactElement, ReactNode } from 'react'
+import { StatusBar, StatusBarStyle } from 'react-native'
+import styled from 'styled-components/native'
+
+import theme from '../constants/theme'
+
+interface Props {
+  backgroundColor?: string
+  barStyle?: StatusBarStyle
+  children?: ReactNode
+  separated?: boolean
```

Comment:
hm yeah i thought about better wording...fullscreen is not better. true sets `flex:0` to avoid that the safe area view also applies to the bottom (color issues)

Reply:
shouldApplyToBottom ?

Reply:
Could you explain a little bit more please? When is this used and why? Which color issues?

Reply:
we have f.e. the `ExerciseFinishedScreen` that has a green top part even for the notification bar but the bottom is gray so if you wrap it with all the content of the screen, the bottom part will appear green. for this case we need 2 separated safeAreaViews (green & gray)

Reply:
Wouldn't it make more sense to be able to pass two separate colors `statusBarColor` and `bottomColor` or similar?

Reply:
<img src="https://user-images.githubusercontent.com/37902063/182354185-0f4b3bf4-b7ab-41f7-bd18-3a14b00985dc.png" width="300" height="600" />